coding-concepts.js

```diff
@@ -66,8 +66,8 @@ class Learn {
     this.year = 2022
   }
 }
-var learnStudent = new Learn("George")
+// var learnStudent = new Learn("George")
 // console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The output will George as the value of the key: student followed by the cohort and year.
+// b) Verify and explain: Logged to terminal was, Learn { student: 'George', cohort: 'Bravo', year: 2022 }. The key word new created a new instance of the object and stored it in the variable, learnStudent. The method constructor takes an argument to be passed as the value of the a key.
```

Comment:
Great work, William!  Your answers here and in your participation in class (!!) show that you are grasping the concepts.  I think once you can make your functions dynamic, you'll be in a good spot.